azure-devops-extension/vss-extension.json

```diff
@@ -0,0 +1,30 @@
+{
+  "manifestVersion": 1,
+  "id": "automator-tools",
+  "name": "Automator Tools",
```

Comment:
Is it really Automator Tools? Should it be named like 'Automator Run Build Script' ?

Reply:
Think of an extension like a Task Suite. `Automator Tools` extension contains multiple tasks, one of which is: `Automator Run Build Script`. In our case this is the only task that it contains 😄.